### DISCUSSION TITLE
Focus on Core Editing Features

### DISCUSSION THREAD

**Discussion by ibrahimdevs** [Flutter-Bounty-Hunters/super_editor#1115] [Ideas]
Super Editor is the best rich text editor for Flutter by far, it is good to support task list, image and other alternative features. But I feel that this package should firstly provide basic text editing as most as bug free. 

Flutter's native text edit widget is very limited, and we need a robust editor at least providing simple bold, italic, highlight features. There are lots of edge case issues and new feature requests. But we can fully trust this editor for users can edit text without a problem (like disappear words, can't delete, unwanted deletion of words etc.). Because an editor's fundamental feature is edit plain text without a problem. The other problems and features comes after that. 

Please don't get me wrong, I can't imagine how effort took this awesome package. I'm using this package in a few of my apps. I can able to add feature like highlight a word, in return of this lots of users complaining me it is too glitchy to edit text now. Imho, It shouldn't be like that,  so I want to share my experience. Maybe it helps to get this package better point. 
@matthew-carroll @angelosilvestre @miguelcmedeiros 

Thanks again for great amount of work and helping for flutter community!

**Comment by matthew-carroll** [comment]
@ibrahimdevs I'm glad to hear that you're making use of super_editor in multiple projects. We have a Discussion post dedicated to companies describing how they're using super_editor, if you'd like to add something.

WRT quality of the experience - the best way to get bugs resolved is to file ticket with very clear reproduction instructions, a minimal code reproduction, and info about your version of Flutter, super_editor, and the platform you're running on.

The question isn't whether we add features OR we make super_editor stable. Instead, we need to find a way to do both. I know there are plenty of issues in the tracker, but in general, we haven't heard about super_editor losing quality, or breaking in obvious ways. If that's happening, please make sure that relevant issues are filed. That's the only way for us to know that you're running into problems.

I'm going to close this issue because there's no specific action for us to take to resolve it.
